vnext/Mso/src/dispatchQueue/looperScheduler.cpp

```diff
@@ -44,35 +44,37 @@ LooperScheduler::~LooperScheduler() noexcept {
   for (;;) {
     if (auto self = weakSelf.GetStrongPtr()) {
       if (auto queue = self->m_queue.GetStrongPtr()) {
-        DispatchQueue q{Mso::CntPtr(queue)};
+        auto &taskStarting = self->m_settings.TaskStarting;
+        auto &taskCompleted = self->m_settings.TaskCompleted;
         DispatchTask task;
         while (queue->TryDequeTask(task)) {
-          if (auto &func = self->m_settings.TaskStarting) {
-            func(q);
+          if (taskStarting) {
+            taskStarting(DispatchQueue{Mso::CntPtr(queue)});
           }
-
           queue->InvokeTask(std::move(task), std::nullopt);
```

Comment:
>queue [](http://example.com/codeflow?start=10&length=5)

nit: please keep the empty lines before and after this line.

Reply:
Done!